modules/meta.analysis/R/jagify.R

```diff
@@ -33,16 +33,16 @@ jagify <- function(result, use_ghs = TRUE) {
   
   r <- PEcAn.DB::assign.treatments(r)
   r <- PEcAn.utils::summarize.result(r)
-  r <- subset(transform(r, 
-                        stat = as.numeric(stat), 
-                        n = as.numeric(n), 
-                        site_id = as.integer(factor(site_id, unique(site_id))), 
-                        greenhouse = as.integer(factor(greenhouse, unique(greenhouse))),
-                        mean = mean, 
-                        citation_id = citation_id,
-                        ghs = greenhouse,
-                        site = site_id,
-                        trt_name = name), 
+  r <- subset(dplyr::mutate(r, 
```

Comment:
If I recall, using 'transmute' instead of 'mutate' will only return the modified columns, making the following 'select' unnecessary.

Reply:
Also, it is not clear - does 'subset' do anything here?